app/src/i18n/locales/en-US.json

```diff
@@ -51,6 +51,12 @@
   "cmps.loop.swap.SwapConfigStep.heading": "{{type}} Amount",
   "cmps.loop.swap.SwapConfigStep.loopInDesc": "Loop In moves funds onto the Lightning Network from an on-chain wallet to provide outbound capacity to send funds over Lightning. Use the slider to select the amount to Loop In or Loop Out of the Lightning Network",
   "cmps.loop.swap.SwapConfigStep.loopOutDesc": "Loop Out moves funds off of the Lightning Network into an on-chain address while providing inbound capacity to receive funds on Lightning. Use the slider to select the amount to Loop In or Loop Out of the Lightning Network",
+  "cmps.loop.swap.SwapConfigStep.confTargetLabel": "Confirmation Target",
+  "cmps.loop.swap.SwapConfigStep.confTargetHint": "number of blocks (ex: 6)",
+  "cmps.loop.swap.SwapConfigStep.destAddrLabel": "Destination Address",
+  "cmps.loop.swap.SwapConfigStep.destAddrHint": "p2wkh address",
```

Comment:
Loop allows any kind of address, even legacy ones. Maybe we should just write `segwit address` to make it a bit less specific but still incentivize people to use native segwit addresses?

Reply:
Good point! I wasn't sure what to put as a placeholder there. `segwit address` makes more sense so I'll switch to that.

I also agree about the tooltip. I'll add one for conf target as well since it likely isn't obvious for new users, what this does.

Thanks for the feedback.